stories/data.js

```diff
@@ -74,7 +74,7 @@ const largeDataset = [
   {
     'id':13,
     'first_name':'Sher',
-    'last_name':"O'Fogarty",
+    'last_name':'O\'Fogarty',
```

Comment:
I think `quotes: ["single", { "avoidEscape": true }]`  is good for a "more natural" compromise to keep mostly single quotes, but not resort to readability hurdles like escape backslash.

Reply:
we can apply that to snoke as well

Reply:
At some point we should have a common repo for eslint rc so we can have a common baseline.